JsonSubTypes/JsonSubtypesConverter.cs

```diff
@@ -47,7 +47,19 @@ internal class JsonSubtypesConverter : JsonSubtypes
             _addDiscriminatorFirst = addDiscriminatorFirst;
             foreach (var type in _subTypeMapping)
             {
-                _supportedTypes.Add(type.Value, type.Key);
+                if (_supportedTypes.ContainsKey(type.Value))
```

Comment:
This kind of check is better suited in `JsonSubtypesConverterBuilder` (this class is internal)

Reply:
ConverterBuilder contains inverted dictionary, but we can add check for it anyway